### ISSUE TITLE
Input validation missing

### ISSUE THREAD

**Issue by mathiasbrenner** [AndresCL/sherpany_demo#1]
Input validation is missing, e.g. when I click Allocate without any documents there is no error displayed.

**Comment by AndresCL** [comment]
I have added a rule to disable button when no file is present. Also I added a button to remove all files from dropzone. Thanks for pointing it.

![captura de pantalla 2017-03-08 a las 11 05 40](https://cloud.githubusercontent.com/assets/1790792/23707170/1d146406-03f0-11e7-8698-48a25e4bef2e.png)
![captura de pantalla 2017-03-08 a las 11 11 28](https://cloud.githubusercontent.com/assets/1790792/23707171/1d196ee2-03f0-11e7-905b-688ec273f65d.png)
